src/style.css

```diff
@@ -1 +1,64 @@
 /* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+* {
+  margin: 0;
+  padding: 0;
+}
+
+ul {
+  list-style: none;
+}
+
+a {
+  text-decoration: none;
+}
```

Comment:
<img width="1094" alt="image" src="https://user-images.githubusercontent.com/99652389/179199103-f9432b73-dd0d-40f5-95d0-64069ef829c9.png">
this is comment to 13th and 9th lines